CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
 
 ### Minor
 
+- Text: Remove prop `leading` and related css properties (#784)
```

Comment:
@AlbertCarreras we should only keep this line and remove the `Docs` specific one.

Reply:
DONE